<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccount</class>
 <widget class="QWidget" name="CreateAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>password : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createAccountButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create the Account </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>User name :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
